### ISSUE TITLE
a11y: no label on the search field

### ISSUE THREAD

**Issue by dstorey** [smashingmagazine/redesign#111]
The search field (opened when clicking topics button) doesn't have a label, only a placeholder. This will technically work in modern browsers that are following HTML AAM as placeholder is one of the lowest fallbacks, but may not work in some browsers or screen readers, especially older ones. 

Better to add aria-label="Search Smashing" to make sure.

While you are at it, making it type="search" instead of text will give it a localized control type of search input, giving further context to screen reader users.

**Comment by indysigner** [comment]
This issue was moved to smashingmagazine/smashing-magazine#586
